flink-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/JDBCUpsertTableSinkITCase.java

```diff
@@ -211,4 +213,31 @@ public void testAppend() throws Exception {
 				Row.of(20, 6, Timestamp.valueOf("1970-01-01 00:00:00.02"))
 		}, DB_URL, OUTPUT_TABLE2, new String[]{"id", "num", "ts"});
 	}
+
+	@Test(expected = JobExecutionException.class)
+	public void testTableNotExists() throws Exception {
```

Comment:
why do you test "table not exists" for fixing "multiple flushing not work" issue ?

Reply:
Hi，@libenchao 
As this issue describe，this test will hang rather than throw an Exception when the `parameter connector.write.max-retries > 1`， so I add this case to check that exception will throw properly after retry 3 times.

Reply:
get it.